### ISSUE TITLE
no notification or calendar entry for local user invitees

### ISSUE THREAD

**Issue by brianjmurrell** [owncloud/core#25444]
### Steps to reproduce
1. Create a calendar event and invite a local (to OC) user (which we will call the invitee)
### Expected behaviour

Invitee should either get an ICS e-mail or should see a new event in their calendar.
### Actual behaviour

Neither of the above, a.k.a. nothing at all happens for the invitee.
### Server configuration

**Operating system**:
CentOS Linux release 7.2.1511 (Core)

**Web server:**
Apache 2.4.6

**Database:**
Maria DB 5.5.44

**PHP version:**
PHP 7.0.4

**ownCloud version:** (see ownCloud admin page)
Owncloud 9.0.0

**Updated from an older ownCloud or fresh install:**
8.0, 8.1, 8.2

**Where did you install ownCloud from:**
http://download.owncloud.org/download/repositories/stable/CentOS_7

**Signing status (ownCloud 9.0 and above):**

```
Integrity Check:
Only two additional php scripts added by me. No changes on the owncloud code base.

Results
=======
- core
    - EXTRA_FILE
        - dyndns/update.php
        - adminer/index.php
```

**List of activated apps:**

```
Enabled:
  - activity: 2.2.1
  - calendar: 1.0
  - comments: 0.2
  - contacts: 1.1.0.0
  - dav: 0.1.5
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - encryption
  - external
  - files_external
  - user_external
  - user_ldap
```

**The content of config/config.php:**

```
{
    "system": {
        "instanceid": "ocuu9enp4kko",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "private_ip",
            "public_domain",
            "private_hostname"
        ],
        "datadirectory": "\/var\/ownclouddata",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "overwrite.cli.url": "http:\/\/private_ip\/owncloud",
        "dbtype": "mysql",
        "version": "9.0.0.19",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "myprovider.com",
        "mail_smtpsecure": "tls",
        "mail_smtpport": "587",
        "mail_smtpauth": 1,
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "givenname.lastname",
        "mail_domain": "myprovider.com",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "updatechecker": false,
        "trashbin_retention_obligation": "auto"
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
No

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No
### Client configuration

**Browser:**
Chrome

**Operating system:**
Linux

**Comment by PVince81** [comment]
@DeepDiver1975 @georgehrke duplicate ?

**Comment by georgehrke** [comment]
I don't think so.

@brianjmurrell Can you please be more specific about the reproduction steps?
What do you type into the attendee field?

**Comment by brianjmurrell** [comment]
1. Click on a day in the calendar
2. Add a title
3. Click More ...
4. Under Attendees start typing the name of an OC user
5. Choose the one you want to invite
6. Click Create at the bottom

I will admit that when I start typing a name it looks like it's looking in my address book, not at local OC user accounts and in the case of the local OC user I am trying to invite I get two entries for the same person in the drop-down with no indication what e-mail address is actually behind each one.

Ah ha!  If I enter the e-mail address of the local OC user rather than just choosing one of the matches in the drop down (or even enter enough of the e-mail address so that there is only one in the list and then choose it from the list) then things seem to work.  That local OC user gets both an entry in their calendar as well as an ICS e-mail.

So the problem here seems to be lack of transparency and/or ambiguity of the matches shown in the drop-down list.  Some suggestions (any or all would be an improvement IMHO):

1. Perhaps the mouse-over ought to show the e-mail address rather than repeating the text in the entry of the drop-down.
2. OC users should be shown separately from entries in the address book and/or differently, perhaps with a different background colour or something.
3. Entries in the list should be sorted in the order of
  -   OC local users first
  -   address book entries matching local OC users next
  -   all others.

**Comment by brianjmurrell** [comment]
Were my reproduction steps helpful?

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).)

**Comment by brianjmurrell** [comment]
Why does this "closing as stale" BS keep happening?  This issue has already been closed stale and reopened once before.  Just because an issue doesn't have the resources to work on it right away doesn't mean it's of no value.  Particularly since I have described the issue twice and nobody even responded to my last query.

Can we please knock off those "closing stale issue" crap already?  Project-wide I mean, not just on this issue.

And can somebody please reopen this issue?

**Comment by phil-davis** [comment]
Use of "the bot" is now under discussion! Some way to mark/label issues as "confirmed issue - bot must ignore" or stop "the bot". One of those general resolutions should happen "real soon now" (tm)

Reopening this.

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).)

**Comment by brianjmurrell** [comment]
> Use of "the bot" is now under discussion! Some way to mark/label issues as "confirmed issue - bot must ignore" or stop "the bot". One of those general resolutions should happen "real soon now" (tm)

Any progress here?  "real soon now" was Jan 29.

**Comment by PVince81** [comment]
@DeepDiver1975 @Julian1998 have we addressed this recently ?

**Comment by Julian1998** [comment]
@PVince81 I don't think so! We fixed some stuff with for the shared ics file but we didn't fix anything in the appearing list when choosing an invitee.
